index.html

```diff
@@ -161,6 +161,23 @@ <h2>About <br> Myself</h2>
                <ul class="info"><a href="#">Skills</a></ul>
             </section>
         </section>
+<!-- -------------------------------------------------contact form----------------------------  -->
+        <section class="contact-form">
+            <div class="form-container">
+                <div class="margin-vertical-50px title-container">
+                    <h3 class="text-white contact-me">Contact me</h3>
+                    <p class="text-offwhite text-justified-center font-size-20px" >If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+                </div>
+                <form action="#">
+                    <input type="text" name="full-name" id="name" maxlength="30" placeholder="Name">
+                    <input type="email" name="Email" id="email" required placeholder="me@gmail.com">
+                    <textarea name="message" id="textarea" cols="30" rows="10" placeholder="Write your message here" maxlength="500"></textarea>
+                    <div>
+                        <a href="#" class="button">Get It Touch</a>
+                    </div>
```

Comment:
- [ ] This suppose to be a button element I guess, the form action value will take care of the data submission. The `#` will always redirect you to the homepage. ℹ️ 
- [ ] Please don't forget to add button interaction as well (Hover, enable). This will make your app to be more user friendly. 😄 

> Kindly don't forget to add type attribute to your button. 😄 You can recheck the requirement for confirmation.